pkgs/tools/security/peepdf/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, python3Packages }:
+
+with lib;
+
+let
+  pythonaes = python3Packages.buildPythonPackage rec {
+    pname = "pythonaes";
+    version = "1.0";
+    src = python3Packages.fetchPypi {
+      inherit pname version;
+      sha256 = "0rwzmwdfslzkabvsxgcg630x27favh1pdwc3dyhcpf006p033pbi";
+    };
+  };
+in python3Packages.buildPythonApplication rec {
+  pname = "peepdf";
+  version = "0.4.2";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "1sblv9dsfvh773qmjpxpzlcxibms6kq2x0rkyh55q8njxh16v1kx";
+  };
+
+  postPatch = "sed -i 's/==/>=/' setup.py";
```

Comment:
Can you add a comment about what this is for?

Reply:
@Ekleog it's to relax the version requirements. In nixpkgs we don't have the exact version of the dependencies.